README.md

```diff
@@ -37,7 +37,17 @@ You can use this repo as a template to start your own project.
    and then visit `http://localhost:8000` to see it in action.
 
 That is all. Once you are familiar with this project's structure,
-you can modify it by changing
-the content inside `<div id="main" ...>...</div>` from `index.html`
+you can modify it by changing the content between
+`<!-- The real project content starts from here -->` and `<!-- The real project content ends here -->`
+in `index.html`,
```

Comment:
@iansparks , I believe this is what you mean by "the README need a bit of updating"

Reply:
It says 

```
...Once you are familiar with this project's structure, you can modify it by changing the content inside <div id="main" ...>...</div> from index.html and content inside main.py."
```
But you remove id="main" from the div?

Reply:
Yes, and this latest commit removed that line. I know you said you are no longer a frequent github user nowadays :-)  but if you are reading [this PR from the gitub web ui](https://github.com/rayluo/brython-project-template/pull/3/files), it shows the deleted lines in red.

So, I guess you are satisfied with these finishing touch on this PR?

Reply:
Yes, go for it.